### ISSUE TITLE
Update Gallery with xbatcher ML Cookbook

### ISSUE THREAD

**Issue by cmdupuis3** [ProjectPythia/cookbook-gallery#119]
### Root Repository Name

cmdupuis3/xbatcher-ML-1-cookbook

### Did you check if your contribution would be better suited as an addition to an existing Cookbook, before generating a new one?

yes

### Additional comments

Confused on what I'm supposed to pull here, so this might take a few attempts.

**Comment by dcamron** [comment]
Hey @cmdupuis3! Thanks for starting here. Confusion is normal, we're still ironing out the contributor's guide updates to make this smoother. If you're ready for your cookbook to be published, then our next steps are to transfer your repository over to the ProjectPythia organization, where you'll still be the primary maintainer of the content. If you're ready to start this process, I can point you in the right direction.

**Comment by cmdupuis3** [comment]
@dcamron Great, so the problem I have is that the contributor's guide says to create a branch, and later says to start a pull request, but I'm not sure how the two go together. See [this issue](https://github.com/ProjectPythia/.github/issues/39) for details.

Would it be correct to merge my branch into my fork's main?

**Comment by dcamron** [comment]
Section 3 of that guide is more about ways you can integrate the infrastructure provided in our template into your development workflow, where you open PRs against your own cookbook and get some automated checks and previews to run on your code changes. For _publishing_ your cookbook, we have to move on to section 4 of the contributor's guide, where we transfer your repository to live inside of this ProjectPythia organization. I can help you start that process.

**Comment by cmdupuis3** [comment]
Actually, I would need to be added to the ProjectPythia contributors. Can you do that?

**Comment by dcamron** [comment]
Looks like you accepted the invite and transferred your repo successfully to ProjectPythia/xbatcher-ML-1-cookbook. Take a minute to update your JupyterBook's `_config.yml` with appropriate title, authorship, description, domain/package tags, and optionally a thumbnail. These will be used in generating the gallery card for your cookbook. Then, all you need to do is edit your original post at the top here to reference only `xbatcher-ML-1-cookbook` under Root Repository Name. Once you make that edit, the bot should go open a PR for you to publish your cookbook!

**Comment by cmdupuis3** [comment]
Sounds good! 

So if I want to rerun the bot, I just edit the OP? Or is there some command for that or something?

**Comment by dcamron** [comment]
Looks like you got it! You can check the preview for your gallery entry over in that PR now.

**Comment by cmdupuis3** [comment]
Cool! Clicking on the preview though, I mostly see stuff from the template repo. Is that normal?

**Comment by brian-rose** [comment]
> Cool! Clicking on the preview though, I mostly see stuff from the template repo. Is that normal?

No! We should be able to see the actual cookbook in the preview. @dcamron any idea what went wrong there?

**Comment by dcamron** [comment]
We should move that step earlier in the guide, and it's set appropriately already for your repo. That appears to be a limitation of being designated an "outside collaborator" instead of an organization member. I'll take a look at that, but it should be good for now.

Separately, I don't see anything wrong with the preview in the PR myself?

**Comment by cmdupuis3** [comment]
I think it updated or something... before, you'd see my thumbnail on the gallery page, but on clicking it you'd see the template.
